modular_zubbers/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/ammo/pistol.dm

```diff
@@ -0,0 +1,7 @@
+/obj/projectile/bullet/c35sol
+	damage = 20
+	wound_bonus = -10
+	bare_wound_bonus = 0
+
+/obj/projectile/bullet/c35sol/ripper
+	damage = 12
```

Comment:
```suggestion
	damage = 13
```

Reply:
ehhhhhh... given the wound chance, i still want the damage on this one to be a bit more on the lower side, so I want to see how 12 works out for it first.

Reply:
agreed